test/system/220-healthcheck.bats

```diff
@@ -34,14 +35,15 @@ function _check_health {
 
 @test "podman healthcheck" {
     _build_health_check_image healthcheck_i
-
-    # Run that healthcheck image.
-    run_podman run -d --name healthcheck_c \
+    run_podman run -d --name healthcheck_c             \
                --health-cmd /healthcheck   \
-               --health-interval 1s        \
-               --health-retries 3          \
-               --health-on-failure=kill    \
+               --health-interval 1s                    \
+               --health-retries 3                      \
+               --health-on-failure=kill                \
+               --health-startup-cmd /home/podman/healthcheck \
```

Comment:
I think this is the bug. `$IMAGE` on this old branch is `20221018`, and that image does not include `/home/podman/healthcheck`.

Solutions:
1. bump TAG in helpers.bash to `20240123` (__may have unintended consequences!!__) or
2. remove this, I don't see health-startup-command being an important part of this test; or
3. dunno, I'm sure there are other options. Good luck.